js/PixelPainter.js

```diff
@@ -0,0 +1,114 @@
+const pixelPainter = document.querySelector("#pixelPainter");
```

Comment:
Try to use newlines and comments to help break your code into "chunks" of code. This makes the code easier to read.